vlogsApp/vlogsApp/Custom Cells/BlogCollectionViewCell.swift

```diff
@@ -60,59 +84,62 @@ class BlogCollectionViewCell: UICollectionViewCell {
-            make.right.equalTo(dateAndTimeLabel.snp.left).offset(-10)
-            make.height.equalTo(50)
+        descriptionStackView.snp.makeConstraints { make in
+            make.left.equalToSuperview().offset(8)
+            make.right.equalToSuperview().offset(8)
+            make.top.equalTo(dateAndTitleStackView.snp.bottom).offset(-100)
+            make.bottom.equalTo(contentView.snp.bottom)
         }
-
+        
+        combinedStackView.snp.makeConstraints { make in
+            make.left.equalToSuperview().offset(8)
+            make.top.equalToSuperview().offset(5)
+            make.right.equalToSuperview().offset(-8)
+            make.bottom.equalTo(contentView.snp.bottom)
+        }
+        
         postImageView.snp.makeConstraints { make in
             make.left.equalToSuperview().offset(5)
             make.top.equalToSuperview().offset(5)
             make.right.equalToSuperview().offset(-5)
-            make.bottom.equalTo(titleLabel.snp.top).offset(-5)
+            make.bottom.equalTo(dateAndTitleStackView.snp.top)
+            make.height.equalTo(150)
         }
-
-        descriptionLabel.snp.makeConstraints { make in
-            make.left.equalToSuperview().offset(5)
-            make.bottom.equalTo(contentView.snp.bottom).offset(1)
-            make.right.equalTo(favouritesButton.snp.left).offset(5)
-            make.height.equalTo(100)
-        }
-
-        dateAndTimeLabel.snp.makeConstraints { make in
-            make.left.equalToSuperview().offset(250)
-            make.bottom.equalTo(contentView.snp.bottom).offset(-60)
-            make.right.equalToSuperview().offset(-5)
-            make.height.equalTo(50)
+        
+        dateAndTitleStackView.snp.makeConstraints { make in
+            make.left.equalToSuperview().offset(8)
+            make.right.equalToSuperview().offset(8)
+            make.bottom.equalTo(contentView.snp.bottom)
+            make.top.equalTo(postImageView.snp.bottom)
         }
-
+        
         favouritesButton.snp.makeConstraints { make in
-            make.left.equalToSuperview().offset(250)
-            make.bottom.equalTo(contentView.snp.bottom).offset(0)
-            make.width.equalTo(150)
-            make.height.equalTo(100)
+            make.right.equalToSuperview()
+            make.left.equalToSuperview().offset(300)
+            make.bottom.equalTo(contentView.snp.bottom)
+            make.top.equalTo(descriptionLabel.snp.bottom)
         }
-
-
     }
     
     func blogCellConfigUI() {
         contentView.layer.cornerRadius = 20
         contentView.backgroundColor = UIColor(named: "textFieldColor")
         
         contentView.addSubview(postImageView)
-        contentView.addSubview(titleLabel)
-        contentView.addSubview(descriptionLabel)
-        contentView.addSubview(dateAndTimeLabel)
+        contentView.addSubview(dateAndTitleStackView)
+        contentView.addSubview(combinedStackView)
+        contentView.addSubview(descriptionStackView)
```

Comment:
No need to add 3 stack views , just use one container view and add it as a subview .